### DISCUSSION TITLE
Not a BUG but a FIX on hyperlink CSS

### DISCUSSION THREAD

**Discussion by suchorski** [community/community#58992] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There is a wrong underline when hovering the commit message on the files of a home page of a github project.
![Fix](https://github.com/community/community/assets/5408170/79b81ab9-e5e9-426a-b28e-2065bc296f01)

**Comment by leonden** [comment]
Can you provide any further information for us to properly understand your issue?

**Comment by akash-singh8** [comment]
Yeah, I think it's just to enhance user experience, as it's easy for users to read.

**Comment by jgbenine** [comment]
If you are in a repository within git, this is a pattern used to reference the existence of a link or possible direction. this from the html tag itself... in the case of a repository it is just showing you that you can open this specific commit to better visualize what was done there.

Some browsers set this property as default is a text-decoration: underline, this can be modified but not in this case.
